sdk/core/azure-core-test/src/main/java/com/azure/core/test/policy/RecordNetworkCallPolicy.java

```diff
@@ -112,6 +113,13 @@ public RecordNetworkCallPolicy(RecordedData recordedData) {
             });
     }
 
+    private void redactedAccountName(UrlBuilder urlBuilder) {
+        String[] hostParts = urlBuilder.getHost().split("\\.");
+        hostParts[0] = "REDACTED";
```

Comment:
From what I remember when we are associating requests to responses when using the playback records we explicitly ignore the entire host string. Should be able to simplify the logic to redact the entire host string.

Reply:
That's true. 
My opinion is to leave insensitive data over there as Json file is also a good and quick reference of what the request and response are composed of. Also, it is cheap to have the information over there.

I am flexible of making change to replace all.